Demographics/1. Demographics - Population.R

```diff
@@ -24,7 +24,7 @@ library(broom)
 library(phsstyles)
 
 # Source in global functions/themes script
-#  source("./Master RMarkdown Document & Render Code/Global Script.R")
```

Comment:
I understand why we are simplifying this but wondered why/how come we used to use source("./Master instead of source("Master ?

Reply:
From some Googling, I confirmed that they are functionally the same. The choice is between making it more explicit `./Master...` or 'cleaner' `Master...`, I went for cleaner (that was also the majority preference).

The main issue for me was that we were using both at various points in the project which just makes it more confusing as it implies there's a reason and therefore a difference!